website/blog/2021/08/10/apisix-nginx.md

```diff
@@ -164,13 +164,13 @@ local function init(env)
     local conf_render = template.compile(ngx_tpl)
     local ngxconf = conf_render(sys_conf)
 
-    local ok, err = util.write_file(env.apisix_home .. "/conf/nginx.conf",
+    local ok, err = util.write_file(env.apisix_home . "/conf/nginx.conf",
                                     ngxconf)
```

Comment:
is it a repeated line here? please delete if it is

Reply:
![image](https://user-images.githubusercontent.com/2106987/139654922-ba976c35-520d-438e-a101-8a16920b2694.png)

😄